syntax = "proto3";

import "google/protobuf/empty.proto";

package maid.example;

option cc_generic_services = true;
option py_generic_services = true;

message HelloRequest
{
    string message = 1;
}

message HelloResponse
{
    string message = 1;
}

service HelloService
{
    rpc Hello(HelloRequest) returns(HelloResponse);
    rpc HelloRpc(HelloRequest) returns(HelloResponse);

    rpc HelloNotify(HelloRequest) returns(google.protobuf.Empty);
}
